cores/arduino/main.cpp

```diff
@@ -45,6 +45,9 @@ int main( void )
 
 	for (;;)
 	{
+		// Define by Ethernet library. It is defined as __weak.
+		stm32_eth_scheduler();
```

Comment:
@RickKimball 

I don't know because I didn't study the LwIP stack in case of an "operating system". The NativeEthernet is oriented single threaded core. I refer you to the LwIP wiki or freertos example using the LwIP stack.
Maybe call stm32_eth_scheduler() inside its own thread or process is enough.